applications/FluidDynamicsApplication/python_scripts/projection_utilities.py

```diff
@@ -0,0 +1,128 @@
+from math import floor, ceil
```

Comment:
This file does not contain any Kratos related stuff.

I think a discussion point for this PR is where to place these utilities.